Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:latest
+# This Dockerfile creates https://hub.docker.com/r/insufficientlycaffeinated/bob
+# Currently there's no automatic build set up for this, so changes to this file
+# should be followed by rebuilding the container and pushing it to dockerhub.
+# Luckily that won't happen very often (because changes can be made in Circle
+# faster)
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt -y install \
+    llvm-dev \
```

Comment:
Can we change this to be `llvm-10-dev`? Llvm isn't backwards compatible between versions?